### ISSUE TITLE
Accessibility: Dissolve2 QML animations should be able to be set on/off based on user requirements

### ISSUE THREAD

**Issue by RobBuchananCompPhys** [disorderedmaterials/dissolve#2618]
**Describe the feature / issue**
[See](https://github.com/disorderedmaterials/dissolve/pull/2606#discussion_r4062111395)
